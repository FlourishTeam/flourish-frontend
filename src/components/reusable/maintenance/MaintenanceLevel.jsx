import React from 'react';

function MaintenanceLevel() {
  return (
    <div>
            Maintenance level here.
    </div>
  );
}

export default MaintenanceLevel;

