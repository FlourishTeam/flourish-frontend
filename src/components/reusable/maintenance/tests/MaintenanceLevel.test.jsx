import React from 'react';
import { render, cleanup } from '@testing-library/react';
import MaintenanceLevel from '../MaintenanceLevel';

describe('MaintenanceLevel component', () => {
  afterEach(() => cleanup());

  it('renders MaintenanceLevel', () => {
    const { asFragment } = render(<MaintenanceLevel />);
    expect(asFragment()).toMatchSnapshot();
  });
});
