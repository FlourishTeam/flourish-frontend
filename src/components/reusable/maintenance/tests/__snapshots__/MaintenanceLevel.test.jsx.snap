// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MaintenanceLevel component renders MaintenanceLevel 1`] = `
<DocumentFragment>
  <div>
    N/A
  </div>
</DocumentFragment>
`;
